From 56e9530528835a0ee8eecd1a0093da2a60166b06 Mon Sep 17 00:00:00 2001
From: James McCoy <jamessan@debian.org>
Date: Sun, 18 Dec 2016 10:50:33 -0500
Subject: [PATCH] Only enable --deb-emulate-sbuild for non-backports builds

fbf59be7c5efcfbb6b991b06c9a4db148e331a4a unconditionally added
--deb-emulate-sbuild to @doseoptions, but this breaks detection of
available dependencies for backports builds.

Signed-off-by: James McCoy <jamessan@debian.org>
---
 bin/wanna-build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/bin/wanna-build b/bin/wanna-build
index a9e9b27..dcd7903 100755
--- a/bin/wanna-build
+++ b/bin/wanna-build
@@ -1756,7 +1756,8 @@ sub wb_dose_builddebcheck {
 
     my $args = shift;
     my $architecture=$args->{arch};
-    my @doseoptions = qw(--failures --explain --quiet --deb-emulate-sbuild);
+    my @doseoptions = qw(--failures --explain --quiet);
+    push @doseoptions, '--deb-emulate-sbuild' if $distribution !~ /backports/;
     my $packagefiles = $args->{pkgs};
     my $sourcesfile = $args->{src};
 
-- 
2.11.0

